### PR TITLE
Action list for permission service

### PR DESCRIPTION
**Pull request by normanjaeckel**

### CODE REVIEW

**Comment by ostcar** [comment]
"list_of_speakers.delete":                  "TODO",

**Comment by ostcar** [comment]
There are some implicit rules. We have to document them somewhere.

1. The action name of an "Generic permission check" action consists of two parts separated by a dot. The first part has to be a collection-name.
2. if a "Generic permission check" action has a field "meeting_id" in the payload, it has to be the meeting with the relevant permissions.
3. If a "Generic permission check" has a field "id" ind the payload, it has to be the id of the collection, that gave the action its name.
4. If the payload does not have the fields "id" or "meeting_id", it can not be a "Generic permission check" action

**Comment by normanjaeckel** [comment]
These rules should be documented in permission repo.

**Comment by ostcar** [comment]
Could you merge this? I use the new form of the health route and would like to checkout the master

File: tests/system/action/list_of_speakers/test_delete.py [hunk 1]
```diff
@@ -1,30 +1,31 @@
-from tests.system.action.base import BaseActionTestCase
+# from tests.system.action.base import BaseActionTestCase
 
+# list_of_speakers.delete is an internal action where this test does not work.
```
Comment (jsangmeister) [inline tests/system/action/list_of_speakers/test_delete.py:3]: Why does this not work? The test should be executed either way, if something changed which makes this test fail, the test should be adjusted accordingly to make it work again...

**Comment by jsangmeister** [comment]
I am not sure I understand why this is needed for the permission service. Does the service call the backend on each request to see which permissions are needed for the action?

**Comment by ostcar** [comment]
No. Not at runtime. The developer calls it at development time to autogenerates values out of it.

[This script](https://github.com/OpenSlides/openslides-permission-service/blob/master/internal/collection/build_autogen_def/main.go) generates [this file](https://github.com/OpenSlides/openslides-permission-service/blob/master/internal/collection/autogen_def.go)

To do this, [just call this](https://github.com/OpenSlides/openslides-permission-service#autogen-actions)

**Comment by jsangmeister** [comment]
Ah, okay, I see. Why did you not want to put this directly into the permission service? Seems like the easier way instead of fetching it from the backend

**Comment by ostcar** [comment]
Would also be possible. But in this case the backend has to be started before the permission service. When we integrate it into the permission service, they are independent.

File: tests/system/action/list_of_speakers/test_delete.py (hunk 1 above)
Comment (normanjaeckel) [reply tests/system/action/list_of_speakers/test_delete.py:3]: We have no usecase at the moment to delete list of speakers via such a call. I had to disable the test to get the permission thing work. See #379

Comment (jsangmeister) [reply tests/system/action/list_of_speakers/test_delete.py:3]: Yes, we have. `list_of_speakers.delete` is called on almost every model when it is deleted via the cascade deletion. So we need tests for it. Please add them back ASAP.

**Pull request merged**